go.mod

```diff
@@ -55,7 +55,8 @@ require (
 	github.com/minio/madmin-go/v3 v3.0.52
 	github.com/minio/minio-go/v7 v7.0.70
 	github.com/minio/mux v1.9.0
-	github.com/minio/pkg/v2 v2.0.19
+	github.com/minio/pkg/v2 v2.0.17
```

Comment:
?

Reply:
Fixed now, I think it was possibly because of changes since the PR was sent.